bin/imds

```diff
@@ -0,0 +1,49 @@
+#!/bin/sh
+
+set -o errexit
+set -o pipefail
+set -o nounset
+
+IMDS_DEBUG="${IMDS_DEBUG:-false}"
+function log() {
+  if [ "$IMDS_DEBUG" = "true" ]
+  then
+    echo >&2 "$1"
+  fi
+}
+
+if [ "$#" -ne 1 ]
+then
+  log "usage: imds API_PATH"
+  exit 1
+fi
+
+# leading slashes will be removed
+API_PATH="${1#/}"
```

Comment:
some usages were like `/latest/meta-data` and some were like `latest/meta-data`. The real IMDS service handles requests like `GET //latest/meta-data`, but the mock service we use in unit testing doesn't. 🥴